### PR TITLE
Fixes UI icons staying open and crawl text

### LINKED ISSUE #6088: UI icons staying open
image

image

Rejoining the game/ after restart when you had the inventory open, has the icons still there when you rejoin even though it is closed.

Opening and closing does remove it, seems like its an issue to do with switching the UI off.

### PR DESCRIPTION
**Pull request by Semuca**
### Purpose
Fixes #6088 and adds crawl text (One of the health rework issues).

### Notes:
In UI_ItemSlot.cs, I've moved some duplicate code into it's own function and added a test that checks if the item slot is hidden or not before displaying the placeholderImage, which caused those 'phantom icons' to appear.

### Changelog:
CL: Fixed UI icons staying open
CL: Added standing up text

### CODE REVIEW

File: UnityProject/Assets/Scripts/UI/Systems/MainHUD/UI Bottom/ControlIntent.cs [hunk 1]
```diff
@@ -55,7 +56,14 @@ public void OnClickRest()
...
 		clientResting = !clientResting;
-		Chat.AddExamineMsgToClient(restMessage);
+		if (clientResting == true)
+		{
+			Chat.AddExamineMsgToClient(startRestMessage);
+		}
+		else
+		{
+			Chat.AddExamineMsgToClient(endRestMessage);
+		}
```
Comment (ThatDan123) [inline UnityProject/Assets/Scripts/UI/Systems/MainHUD/UI Bottom/ControlIntent.cs]: `Chat.AddExamineMsgToClient(clientResting ? startRestMessage : endRestMessage);`

Cleaner as you're not needing to do anything else in the IF

**Comment by PerfectTangent** [comment]
`added a test that checks if the item slot is hidden or not before displaying the placeholderImage, which caused those 'phantom icons' to appear.`

hell YES!

**Comment by Semuca** [comment]
This is ready to merge btw

**Comment by NoooneyDude** [comment]
It isn't now, unfortunately ; ). Merge conflicts - just overwrite my changes and then re-namespace 'em.

**Comment by PerfectTangent** [comment]
does it fix this line on the health issue?

`when you try to get up after crawling, it displays the same text "You try to lie down."`

**Comment by Semuca** [comment]
> It isn't now, unfortunately ; ). Merge conflicts - just overwrite my changes and then re-namespace 'em.

Ok, I'll get to that

> does it fix this line on the health issue?
> 
> `when you try to get up after crawling, it displays the same text "You try to lie down."`

Yup

**Comment by Semuca** [comment]
Alright, I think this is all clear and ready to merge

**Comment by Semuca** [comment]
Agh, I surrender to Git. I need some sleep, I'll return tomorrow.

**Comment by Semuca** [comment]
I'm not the best at git, but the shenanigans are over now

**Pull request merged**